common/sai_client/sai_client.py

```diff
@@ -0,0 +1,96 @@
+import sys
+import os
+
+sys.path.insert(0, os.path.dirname(os.path.dirname(__file__)))
+
+class SaiClient:
+    """SAI client interface to wrap low level SAI calls. Is used to define own SAI wrappers"""
+    def __init__(self, client_config):
+        raise NotImplementedError
+
+    def cleanup(self):
+        raise NotImplementedError
+
+    def set_loglevel(self, sai_api, loglevel):
+        raise NotImplementedError
+
+    # CRUD
+    def create(self, obj_type, *, key=None, attrs=None, do_assert=True):
```

Comment:
I think `*, ` here is excessive. Due to this, the TCs will be polluted with params names. Please remove here and below.